chapters/ch53.md

```diff
@@ -1,61 +1,61 @@
 > # 53. Error analysis by parts
 
---> _replace THIS LINE by your translation for the above line_
+#53. Phân tích lỗi từng phần
 
 > Suppose your system is built using a complex machine learning pipeline, and you would like to improve the system’s performance. Which part of the pipeline should you work on improving? By attributing errors to specific parts of the pipeline, you can decide how to prioritize your work.
 
---> _replace THIS LINE by your translation for the above line_
+Giả sử như hệ thống của bạn được xây dựng dựa trên một pipeline học máy phức tạp và bạn muốn cải thiện chất lượng của nó. Bạn nên cải tiến phần nào trong pipeline này? Bạn có thể lên kế hoạch ưu tiên những việc cần làm để cải thiện bằng cách xác định độ lỗi của từng phần trong pipeline. 
 
 > Let’s use our Siamese cat classifier example:
 
---> _replace THIS LINE by your translation for the above line_
+Hãy sử dụng ví dụ bộ phân loại mèo Xiêm của chúng ta:
 
 ![img](../imgs/C53_01.png)
 
 > The first part, the cat detector, detects cats and crops them out of the image. The second part, the cat breed classifier, decides if it is a Siamese cat. It is possible to spend years working on improving either of these two pipeline components. How do you decide which component(s) to focus on?
 
---> _replace THIS LINE by your translation for the above line_
+Phần đầu tiên, bộ phát hiện mèo, sẽ phát hiện ra mèo và cắt chúng ra khỏi tấm ảnh. Phần thứ hai, bộ phân loại giống mèo, xác định xem đó có phải thuộc giống mèo Xiêm không. Việc cải thiện bất kì bộ phận nào trong pipeline này cũng có thể tốn tới hàng năm trời. Làm sao để bạn quyết định được (những) bộ phận nào cần tập trung cải thiện?
 
 > By carrying out ​**error analysis by parts​**, you can try to attribute each mistake the algorithm makes to one (or sometimes both) of the two parts of the pipeline. For example, the algorithm misclassifies this image as not containing a Siamese cat (y=0) even though the correct label is y=1.
 
---> _replace THIS LINE by your translation for the above line_
+Bằng việc thực hiện *phân tích lỗi từng phần*, bạn có thể cố quy trách nhiệm cho một (hoặc đôi khi là cả hai) phần trong pipeline trên từng dự đoán sai của thuật toán.
 
 ![img](../imgs/C53_02.png)
 
 > Let’s manually examine what the two steps of the algorithm did. Suppose the Siamese cat detector had detected a cat as follows:
 
---> _replace THIS LINE by your translation for the above line_
+Hãy cùng xem xét thủ công thuật toán đang làm gì theo từng bước. Giả sử bộ phát hiện mèo Xiêm đã phát hiện ra một chú mèo như dưới đây: 
```

Comment:
* "thủ công thuật toán" nghe hơi dị. Đề xuất:
"Hãy kiểm chứng một cách thủ công hai bước của thuật toán đã làm gì.

Reply:
sửa thành: Hãy kiểm chứng một cách thủ công xem hai bước của thuật toán đã làm gì